src/flatfile.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2019 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_FLATFILE_H
+#define BITCOIN_FLATFILE_H
+
+#include <chain.h>
```

Comment:
Commit 88cf780

Prefer forward declaration of `CDiskBlockPos`?

Reply:
Good point. Though CDiskBlockPos is moved into this header later on, so I don't think it's necessary to change in the earlier commit.